src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public abstract class Machine {
+    public abstract void doWork();
```

Comment:
no need to write abstract , it's an interface

Reply:
![Screenshot_10](https://user-images.githubusercontent.com/49286476/194146158-d6b8377e-c8fe-4f0d-ad2a-3ec8859cbfb6.png)

Reply:
Yes, you're right, sorry for misunderstanding